emData/download.sh

```diff
@@ -11,6 +11,12 @@ luts_url_reduced="https://cernbox.cern.ch/index.php/s/GJZA1zLnWg3hP4y/download"
 # Combined modules
 memprints_url_cm="https://cernbox.cern.ch/index.php/s/YcqX3KUgFdZyMG6/download"
 luts_url_cm="https://cernbox.cern.ch/index.php/s/lKrxzKJ0XmelE0j/download"
+# Reduced Combined modules                                                                                                                     
+#memprints_url_reducedcm="https://aryd.web.cern.ch/aryd/MemPrints_CombinedReduced_220309.tgz"
+#luts_url_reducedcm="https://aryd.web.cern.ch/aryd/LUTs_CombinedReduced_220309.tgz"
+#memprints_url_reducedcm="https://aryd.web.cern.ch/aryd/MemPrints_CombinedReduced_220411.tgz"
+memprints_url_reducedcm="https://aryd.web.cern.ch/aryd/MemPrints_CombinedReduced_220503.tgz"
+luts_url_reducedcm="https://aryd.web.cern.ch/aryd/LUTs_CombinedReduced_220411.tgz"
```

Comment:
@aryd Can you add a recipe for reproducing these files to the README? There are now various sections for the different configurations, so you can add another section for this configuration.

Reply:
This description has been added. This was tested in CMSSW_12_5_0_pre2. With the code on the branch ReducedModuleConfigurationFixes (PR#182).